library/core/src/num/int_log10.rs

```diff
@@ -1,141 +1,140 @@
-mod unchecked {
-    // 0 < val <= u8::MAX
-    #[inline]
-    pub const fn u8(val: u8) -> u32 {
-        let val = val as u32;
-
-        // For better performance, avoid branches by assembling the solution
```

Comment:
These are unchanged, just de-indented.  You might want to use [ignore whitespace mode](https://github.com/rust-lang/rust/pull/92956/files?w=1) to review.